File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int max_words = INT_MIN;
        for(string sentence : sentences)
        {
            int words = 1;
            for(int i = 0; i < sentence.length(); i++)
            {
                if(sentence[i] == ' ')
                {
                    words++;
                }
            }
            max_words = max(max_words, words);
        }
        return max_words;
    }
};

File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max_words = Integer.MIN_VALUE;
        for (String sentence : sentences) {
            int counted_words = sentence.split(" ").length;
            if (counted_words > max_words) {
                max_words = counted_words;
            }
        }
        return max_words;
    }
}